_posts/0000-01-02-Zenphyrrr.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
```

Comment:
# This is an <h1> tag

Reply:
###### This is an <h6> tag

Reply:
*This text will be italic*

Reply:
*This text will be italic*
_This will also be italic_

**This text will be bold**
__This will also be bold__

_You **can** combine them_

Reply:
* Item 1
* Item 2
  * Item 2a
  * Item 2b

Reply:
1. Item 1
1. Item 2
1. Item 3
   1. Item 3a
   1. Item 3b

Reply:
1. Item 1
 1. Item 2
  1. Item 3
    1. Item 3a
      1. Item 3b

Reply:
![GitHub Logo](/images/logo.png)
Format: ![Alt Text](url)

Reply:
http://github.com - automatic!
[GitHub](http://github.com)

Reply:
As Kanye West said:

> We're living the future so
> the present is our past.

Reply:
I think you should use an
`<addr>` element here instead.

Reply:
```javascript
function fancyAlert(arg) {
  if(arg) {
    $.facebox({div:'#foo'})
  }
}
```